web/client/components/geostory/layouts/sections/Section.jsx

```diff
@@ -72,6 +74,7 @@ class Section extends React.Component {
                 update={this.props.update}
                 inViewRef={this.props.inViewRef}
                 editMedia={this.props.editMedia}
+                updateSection={(element) => this.props.updateSection(`sections[{"id": "${this.props.id}"}]`, element, "merge")}
```

Comment:
update should be enough. It's already mapped to `sections[{id...}]` by the enhancer

Reply:
in this case i need an update where the path works only for sections